source/server/admin/admin.h

```diff
@@ -308,8 +308,6 @@ class AdminImpl : public Admin,
 
     // Config::ConfigProvider
     SystemTime lastUpdated() const override { return time_source_.systemTime(); }
-    const Protobuf::Message* getConfigProto() const override { return nullptr; }
-    std::string getConfigVersion() const override { return ""; }
```

Comment:
High-level question:
IIUC currently dynamic config providers had a version, but it wasn't exported anywhere. This PR removes that version.
Would it make more sense to attach the version to plumb the version so it will be exported as part of the config in the admin-console?

Reply:
I'm not sure to be honest. IMHO the fact that this hasn't been used at all before somehow explains no one wants it or there's no use case for it.